pom.xml

```diff
@@ -138,10 +147,36 @@
                 </plugins>
             </build>
         </profile>
+        
     </profiles>
 
     <build>
-        <plugins>
+    	
+    	<!-- You may run a build without any goal .. just type 'mvn' -->
+    	<defaultGoal>clean install</defaultGoal>
```

Comment:
Could you provide some examples or other large projects that use this?

For example the Maven Surefire Project only uses this in the context of a profile rather then a project default. This leads me to believe that this is intended for specific uses rather then a generic `mvn` command.

https://github.com/apache/maven-surefire/blob/acc1e7d907803b1de4f6692194500b5487b6dc91/maven-failsafe-plugin/pom.xml#L225-L228

Reply:
I explained the main motivation in the PR. I think it is a good practice if the project is self-documenting the way you build it. The whole PR is honestly based on lack of knowledge of project details and pitfalls that I ran into when I tried to build it for the first time.

Reply:
I'm not looking for a motivation. That is reasonably clear here.

Maven works with conventions. So we can assume people will known the main goals of Maven and their purpose. As such we should try to avoid customizing the setup as much as possible. So what I'm looking for is some evidence that setting `defaultGoal` is a well known convention. If you don't know any major projects that use this, a a subject matter expert (such as a conference speaker) recommending this is fine too.

Reply:
Wait, did you start this PR because `mvn test` currently doesn't work?